Makefile.calico-node

```diff
@@ -240,13 +249,14 @@ st: dist/calicoctl busybox.tar routereflector.tar calico-node.tar workload.tar r
 	           --rm -ti \
 	           -v /var/run/docker.sock:/var/run/docker.sock \
 	           -v $(SOURCE_DIR):/code \
+	           -v $(SOURCE_DIR)/tests:/tests \
```

Comment:
since we're already mounting $(SOURCE_DIR) as /code, do we really need to mount `test` too?